lib/rubocop/cop/style/mixin_grouping.rb

```diff
@@ -36,12 +36,17 @@ class MixinGrouping < Cop
         MIXIN_METHODS = %i[extend include prepend].freeze
         MSG = 'Put `%<mixin>s` mixins in %<suffix>s.'.freeze
 
-        def on_send(node)
-          return unless node.macro? && MIXIN_METHODS.include?(node.method_name)
+        def on_class(node)
+          begin_node = node.child_nodes.find(&:begin_type?) || node
+          begin_node.each_child_node(:send).select(&:macro?).each do |macro|
```

Comment:
It would be more performant to write this as

```ruby
begin_node.each_child_node(:send) do |child_node|
  next unless child_node.macro?
  ...
end
```

This would likely have minimal real world impact so don't feel obligated to make the change.

Reply:
I'm fine with the current code, as it reads better and I don't think it will impact performance much.